eventuals/task.h

```diff
@@ -339,7 +352,8 @@ struct _TaskFromToWith final {
               HAS_ARGS,
               std::true_type,
               std::is_invocable<F>>::value,
-          "'Task' expects a callable (e.g., a lambda) that takes no arguments");
+          "'Task' expects a callable (e.g., a lambda) "
+          "that takes no arguments");
```

Comment:
This formatting change looks unnecessary, can we please revert?

Reply:
exceed 80 characters

Reply:
How did we get it committed before!? It doesn't look like it to me!